mod/file/thumbnail.php

```diff
@@ -6,7 +6,12 @@
  */
 
 // Get engine
-require_once __DIR__ . '/../../vendor/autoload.php';
+$initialRoot = dirname(dirname(__DIR__));
+$backupRoot = dirname(dirname(dirname($initialRoot)));
+
+if (!include_once "$initialRoot/vendor/autoload.php") {
```

Comment:
I didn't give this any thought because I copied it directly from the profile plugin.